mne/epochs.py

```diff
@@ -1491,6 +1493,86 @@ def get_data(self, picks=None, item=None):
         """
         return self._get_data(picks=picks, item=item)
 
+    @verbose
+    def apply_function(self, fun, picks=None, dtype=None, n_jobs=1,
+                       channel_wise=True, verbose=None, *args, **kwargs):
+        """Apply a function to a subset of channels.
+
+        The function ``fun`` is applied to the channels defined in ``picks``.
+        The data of the Epochs object is modified in-place. If the function
+        returns a different data type (e.g. numpy.complex) it must be specified
+        using the dtype parameter, which causes the data type used for
+        representing the raw data to change.
```

Comment:
there is a lot here that is redundant with the method you're adding to Evokeds, and redundant with the existing docstring for `Raw.apply_function()`.  Also the docstring parameters are now copy-pasted to 3 places in the codebase, which means changes potentially have to be made 3 times.  This is a perfect opportunity to learn to use our `docdict`!  Start by adding an entry to `mne/utils/docs.py` called `apply_function_summary` or so, that has this paragraph, and then other entries for the repeated function params.  Then come back here and insert them with the `%(docdict_key)s` method (like is already done for `%(picks_all_data_noref)` below).  For an example of how you can factor out the common elements of a paragraph, see the example of how we handle the various forms that the `picks` parameter can take: 

https://github.com/mne-tools/mne-python/blob/c638f74ac727aeaca3b9616e56115b990a84c7ba/mne/utils/docs.py#L439-L463

Reply:
Great! I have done that now. Also for the `*args` and `**kwargs`. If agree on this, we should also change the raw function for consistency with the epochs and evoked objects?